browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -5708,6 +5708,13 @@ L.CanvasTileLayer = L.Layer.extend({
 		if (this.isCalc() && !this._gotFirstCellCursor)
 			return;
 
+		// be sure canvas is initialized already and has correct size
+		var size = map.getSize();
+		if (size.x === 0 || size.y === 0) {
+			setTimeout(function () { this._update(); }.bind(this), 1);
```

Comment:
How do we know that after 1 second canvas will be initialized? What if it still isn't? 

Can't we trigger the `_update()` call when canvas is initialized the first time?